### DISCUSSION TITLE
Chromixs Rg values across SEC elution curve

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1953] [Primary data processing] [answered]
Hi,

A question about Chromixs in ATSAS 4.0.1: when you select your sample frames and buffer frames from your SEC elution trace and then calculate structural parameters, the software only calculates the Rg values for the selected sample frames (rather than for the whole peak). Is there a way to calculate the Rg values for the whole peak as I would like to show the Rg trend across the whole peak as well as for just the sample frames I have selected to merge.

Many thanks.

**Comment by franke-biosaxs** [comment, marked as answer]
Chromixs always used only the selected sample frames to do that. In previous versions one had to change the frame selection to get a wider view of the RG values. Now you can right click on a Frame Selection you have, click "Duplicate Selection", then add sample frames to the left and right of your peak. Calculate the structural parameters of this second selection. Result: a wider view of the Rg distribution with your original selection unchanged.

**Reply by I-G-E** [reply]
Thank you very much, that makes sense. Then I can view the Rg distribution across the entire peak, but only use my smaller selection of frames to merge for data processing.
